.travis.yml

```diff
@@ -22,7 +22,7 @@ matrix:
     - php: nightly
 
 before_install:
-  - phpenv config-rm xdebug.ini
+  - phpenv config-rm xdebug.ini || true
```

Comment:
Can you point me to a failure? I can't seem to find one related to this change?

Reply:
The thing is Xdebug is not available for PHP 8 yet so travis build was failing here. Suppressing it with `|| true` allows script to continue if Xdebug was not installed. I added `--ignore-platform-reqs` flag to composer to let it install the dependencies on upcoming PHP version but the build is still failing because of the deprecation errors. At least we are able to see the errors so we can work on fixing them 😉 .

Reply:
Failures are related to https://github.com/phpspec/phpspec/pull/1313

Reply:
ok, do you want to keep this PR open untill phpspec is fixed or do you want me to merge it in as-is?

Reply:
Merging now shouldn't hurt. We can just keep an eye on nightly CI sometimes to see if it's green because it's most likely only phpspec's problem and should be resolved soon without any changes to this repo.